image/add-gateway.sh

```diff
@@ -166,6 +179,43 @@ main() {
     sudo mv "${ADDONS_DIR}/package" "${ADDONS_DIR}/zwave-adapter"
     rm -rf "${tempdir}"
 
+    # Setup things so that the filesystem gets resized
+    # on the next boot.
+
+    sudo sh -c "cat > '${ETC_DIR}/init.d/resize2fs_once'" <<'END'
+#!/bin/sh
+### BEGIN INIT INFO
+# Provides:          resize2fs_once
+# Required-Start:
+# Required-Stop:
+# Default-Start: 3
+# Default-Stop:
+# Short-Description: Resize the root filesystem to fill partition
+# Description:
+### END INIT INFO
+. /lib/lsb/init-functions
+case "$1" in
+  start)
+    log_daemon_msg "Starting resize2fs_once"
+    ROOT_DEV=$(findmnt / -o source -n) &&
```

Comment:
Don't you need trailing line continuation characters `\` on the next several lines after `&&`?